test/runtests.jl

```diff
@@ -3,7 +3,11 @@ using Base.Test
 
 tic()
 @time @testset "Gillespie Tests" begin include("gillespie.jl") end
+@time @testset "Run Another Gillespie Tests" begin include("test_gillespie.jl") end
```

Comment:
No, keep the file but don't run it. Maybe it should move to benchmarking repos later, but for not it's here.